pkg/collect/ceph.go

```diff
@@ -105,23 +106,65 @@ var CephCommands = []CephCommand{
 	},
 }
 
-func Ceph(c *Collector, cephCollector *troubleshootv1beta2.Ceph) (CollectorResult, error) {
+type CollectCeph struct {
+	Collector    *troubleshootv1beta2.Ceph
+	BundlePath   string
+	Namespace    string
+	ClientConfig *rest.Config
+	Client       kubernetes.Interface
+	ctx          context.Context
+	RBACErrors   []error
+}
+
+func (c *CollectCeph) Title() string {
+	return collectorTitleOrDefault(c.Collector.CollectorMeta, "Cluster Info")
+}
+
+func (c *CollectCeph) IsExcluded() (bool, error) {
+	return isExcluded(c.Collector.Exclude)
+}
+
+func (c *CollectCeph) GetRBACErrors() []error {
+	return c.RBACErrors
+}
+
+func (c *CollectCeph) HasRBACErrors() bool {
+	return len(c.RBACErrors) > 0
+}
+
+func (c *CollectCeph) CheckRBAC(ctx context.Context, collector *troubleshootv1beta2.Collect) error {
+	exclude, err := c.IsExcluded()
+	if err != nil || exclude != true {
+		return nil
+	}
```

Comment:
should this be exclude == true?

```suggestion
	exclude, err := c.IsExcluded()
	if err != nil || exclude {
		return nil
	}
```